src/Catalyst.Core.Modules.Dfs/LinkedData/IpldRegistry.cs

```diff
@@ -0,0 +1,69 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using System.Collections.Generic;
+
+namespace Catalyst.Core.Modules.Dfs.LinkedData
+{
+    /// <summary>
+    ///   Metadata on <see cref="ILinkedDataFormat"/>.
+    /// </summary>
+    public static class IpldRegistry
+    {
+        /// <summary>
+        ///   All the supported IPLD formats.
+        /// </summary>
+        /// <remarks>
+        ///   The key is the multicodec name.
+        ///   The value is an object that implements <see cref="ILinkedDataFormat"/>.
+        /// </remarks>
+        public static Dictionary<string, ILinkedDataFormat> Formats;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use an immutable collection or reduce the accessibility of the field(s) 'Formats'.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)